src/webui/components/mcp_server_config_tab.py

```
@@ -0,0 +1,154 @@
+from src.utils.mcp_utils import test_mcp_connection
+
+logger = logging.getLogger(__name__)
+
+def create_mcp_server_config_tab(webui_manager: WebuiManager):
+    """
+    Creates a tab for configuring MCP servers.
+    """
+    tab_components = {}
+
+    # Load existing MCP server configuration
+    mcp_config = webui_manager.get_config_value("mcp", "mcpServers", {})
```

Ruff (0.11.9)

Local variable `mcp_config` is assigned to but never used

Remove assignment to unused variable `mcp_config`

(F841)